src/projects/_data.js

```diff
@@ -0,0 +1,5 @@
+export function url(page) {
```

Comment:
I don't think this is the right solution. You can find more info [here](https://lume.land/plugins/jsx/)

Reply:
I thought this might have been the issue but we already have jsx configured in the _config.ts

Reply:
https://github.com/officialrajdeepsingh/Minimalist-blog

Reply:
I think this solution is viable since, unless you have something else to refer to @miguel-merlin. Nonetheless we should merge to see if it works on the live site and maybe look back later